test/sequential/test-timers-block-eventloop.js

```diff
@@ -11,7 +11,7 @@ const t2 = setInterval(() => {
   common.busyLoop(15);
 }, 10);
 
-const t3 = setTimeout(common.mustNotCall('eventloop blocked!'), 100);
+const t3 = setTimeout(common.mustNotCall('eventloop blocked!'), 200);
```

Comment:
Nit: it might make sense to use `common.platformTimeout()` everywhere and keep the original values. Not sure if it is enough though.

Reply:
I'd also prefer `common.platformTimeout()` if that works.

Reply:
I tested it with `common.platformTimeout` just now. Behavior is the same with `100`. I will update the case with `200` and `common.platformTimeout` togegher.